build fail looks real
```
======================================================================
ERROR: corehq.form_processor.tests.test_reindex_accessors:UnshardedCaseReindexAccessorTests.test_get_doc_count
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/form_processor/tests/test_reindex_accessors.py", line 119, in test_get_doc_count
    self.assertEqual(16, self.accessor_class().get_approximate_doc_count('default'))
  File "/mnt/commcare-hq-ro/corehq/form_processor/backends/sql/dbaccessors.py", line 315, in get_approximate_doc_count
    query = self.query(from_db, for_count=True)
  File "/mnt/commcare-hq-ro/corehq/form_processor/backends/sql/dbaccessors.py", line 271, in query
    filters = self.filters(last_doc_pk, for_count)
  File "/mnt/commcare-hq-ro/corehq/form_processor/backends/sql/dbaccessors.py", line 267, in filters
    filters.extend(self.extra_filters(for_count=for_count))
  File "/mnt/commcare-hq-ro/corehq/form_processor/backends/sql/dbaccessors.py", line 792, in extra_filters
    if self.start_date is not None:
AttributeError: 'CaseReindexAccessor' object has no attribute 'start_date'
```

--- a/corehq/form_processor/backends/sql/dbaccessors.py
+++ b/corehq/form_processor/backends/sql/dbaccessors.py
@@ -770,6 +770,8 @@ class CaseReindexAccessor(ReindexAccessor):
     def __init__(self, domain=None, limit_db_aliases=None, start_date=None, end_date=None):
         super(CaseReindexAccessor, self).__init__(limit_db_aliases=limit_db_aliases)
         self.domain = domain
+        self.start_date = start_date
+        self.end_date = end_date
 
     @property
     def model_class(self):

